nss-day-cohort-60/rare-client-tenacious-tators
# Lh add post

Closes Ticket #18


# NewPost module added to create a new article

---

# Steps to Test

1. Fetch the `lh-add-post` branch and switch to it
2. Run npm start to open the app in the browser
3. Navigate to Posts and click Add Post
4. Fill out the form: [title: A sweet potato 
imageURL: https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcRZtdfv1QvgDnTjUUyRnnhvgFSDfVLCmk0BxQ&usqp=CAU
content: Article body test content]
6. Verify that when the publish button is clicked, the browser navigates to the Post page and the new article appears.
